src/lints/enum_struct_union_deprecated_added.ron

```diff
@@ -0,0 +1,62 @@
+SemverQuery(
+    id: "enum_struct_union_deprecated_added",
+    human_readable_name: "enum/struct/union #[deprecated] added",
+    description: "Either an enum, a struct or a union has been newly marked with #[deprecated].",
```

Comment:
Enums, structs, and unions are all kinds of types, so let's simplify the terminology.
 ```suggestion
    id: "type_marked_deprecated",
    human_readable_name: "#[deprecated] added on type",
    description: "A type has been newly marked with #[deprecated].",
```

Reply:
If you agree with the suggested rename, please remember to change the name of the file, update `src/query.rs` with the new name, and also change the name of the test crates and test outputs.